File: project-e-commerce-backend/src/main/java/com/example/project_e_commerce_backend/dtos/CartDto.java
package com.example.project_e_commerce_backend.dtos;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class CartDto {
    private Long productId;

    private String name;

    private String size;

    private String color;

    private int quantity;

    public CartDto(Long productId, String name, String size, String color, int quantity) {
        this.productId = productId;
        this.name = name;
        this.size = size;
        this.color = color;
        this.quantity = quantity;
    }
}

File: project-e-commerce-frontend/src/dtos/CartDto.js
export default class CartDto {
    _productId;
    _name;
    _size;
    _color;
    _quantity;


    constructor(productId, name, size, color, quantity) {
        this._productId = productId;
        this._name = name;
        this._size = size;
        this._color = color;
        this._quantity = quantity;
    }

    setQuantity(quantity){
        this._quantity = quantity;
    }
}